e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
Let's not disable ESLint rules globally for the file
I agree, that the second rule is a little strange

If you want to disable this rule - let's do it through the ESLint config :)